### ISSUE TITLE
display date column / filter by date

### ISSUE THREAD

**Issue by globatum** [13pixlar/sticky-list#21]
How can we filter all form entries by a certain time period i.e Jan 1, 2018 to Aug 17, 2018 from a form’s sticky list settings?

second question if the above is not yet possible, how can we display the date and time of the submission in the table created from [stickylist id=1″]?

**Comment by 13pixlar** [comment]
Yes. You can use the filter `filter_entries` to filter out any entries you dont want to show. This example filters out entries based on filed value but you could modify it to filter by date.

`add_filter('filter_entries','hide_some_rows' );
function hide_some_rows($entries) {
    foreach ($entries as $entryKey => $entryValue) {
        if ($entryValue["1"] == "some-text") {
            unset($entries[$entryKey]);
        }
    }
    return $entries;
}`
